velox/exec/PlanNodeStats.h

```diff
@@ -173,5 +173,7 @@ folly::dynamic toPlanStatsJson(const facebook::velox::exec::TaskStats& stats);
 std::string printPlanWithStats(
     const core::PlanNode& plan,
     const TaskStats& taskStats,
-    bool includeCustomStats = false);
+    bool includeCustomStats = false,
+    std::function<std::string(const core::PlanNodeId& planNodeId)> annotation =
```

Comment:
Do you want to give an alias for std::function<std::string(const core::PlanNodeId& planNodeId)> by using?

const T& annotation